install_config/aggregate_logging.adoc

```diff
@@ -1319,19 +1311,20 @@ config.yaml: |
   .defaults:
     delete:
       days: 31
-    runhour: 0
-    runminute: 0
-    timezone: America/New_York
 
   .regex:
     - pattern: '^project\..+\-dev\..*$'
       delete:
-        days: 15
+        days: 1
     - pattern: '^project\..+\-test\..*$'
       delete:
-        days: 30
+        days: 2
 ----
 
+[NOTE]
+====
+The timezone is set based on the {product-title} master node.
+====
 
 [IMPORTANT]
 ====
```

Comment:
When you use `month` as the `$UNIT` for an operation
change to
When you use `months` as the `$UNIT` for an operation

Reply:
Thanks, changed!